integration_tests/test_basic.py

```diff
@@ -242,6 +246,89 @@ def test_statesync(cronos):
     )
 
     print("succesfully syncing")
+    clustercli.supervisor.stopProcess(f"{clustercli.chain_id}-node{i}")
+
+
+def test_local_statesync(cronos):
+    """
+    - init a new node
+    - dump snapshot on node0
+    - load snapshot to the new node
+    - restore the new node state from the snapshot
+    - bootstrap cometbft state
+    - startup the node, should sync
+    - cleanup
+    """
+    # wait for the network to grow a little bit
+    cli0 = cronos.cosmos_cli(0)
+    wait_for_block(cli0, 6)
+
+    sync_info = cli0.status()["SyncInfo"]
+    height = int(sync_info["latest_block_height"])
+    cronos.supervisorctl("stop", "cronos_777-1-node0")
+    # print(cli0.list_snapshot())
+    tarball = cli0.data_dir / "snapshot.tar.gz"
+    cli0.export_snapshot(height)
```

Comment:
height for export seems flaky, do we need fallback height if a more recent snapshot already exists

Reply:
I can make it check the `list_snapshot` first.